change/@ni-spright-components-08e968ac-291f-4c66-949e-115ca77deef2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
waiting for a passing build to verify no regressions in build logs for new infra